tests/FsPickler.PerfTests/paket.references

```diff
@@ -1,7 +1,9 @@
-FSharp.Charting
+FSharp.Core

```

Comment:
This should be under "group Test".  All the different "test" directories are using the same directory ``../../bin``, see https://github.com/mbraceproject/FsPickler/blob/master/tests/FsPickler.PerfTests/FsPickler.PerfTests.fsproj#L22